packages/elements/src/clock/__test__/clock.test.js

```diff
@@ -111,5 +111,14 @@ describe('clock/Clock', () => {
       expect(valueEvent.detail.value, 'event\'s value should be 00:00:01').to.be.equal('00:00:01');
     });
   });
+
+  describe('Tick', () => {
+    beforeEach(async () => {
```

Comment:
Why do you need `beforeEach` here?

Reply:
Updated to remove this because it use only once times.